objc/notify.m

```diff
@@ -40,10 +42,12 @@ BOOL setApplication(NSString* newbundleIdentifier)
 
         NSUserNotificationCenter* notificationCenter = [NSUserNotificationCenter defaultUserNotificationCenter];
         NotificationCenterDelegate* ncDelegate = [[NotificationCenterDelegate alloc] init];
-        // By default, don't wait for actions. This is set to YES when a button/action-related option is set.
-        ncDelegate.keepRunning = NO;
         notificationCenter.delegate = ncDelegate;
 
+        // By default, do not wait for interaction unless an action or schedule is set.
+        // This can be overriden with `asynchronous` in order to always "fire and forget"
```

Comment:
Just realised. This can be overriden wit `synchronous` not async. Or am I mistaken here?